projects/robots/a4/portal/worlds/.portal_pe.wbproj

```diff
@@ -4,5 +4,8 @@ simulationViewPerspectives: 000000ff000000010000000200000000000004e2010000000601
 sceneTreePerspectives: 000000ff0000000100000002000000c0000000db0100000006010000000200
 maximizedDockId: -1
 centralWidgetVisible: 1
+selectionDisabled: 0
+viewpointLocked: 0
 orthographicViewHeight: 1
-textFiles: 0 "controllers/portal/portal.c"
+textFiles: -1
```

Comment:
Is it wanted that the controller file is no longer opened by default?
Otherwise this could be changed to 

```suggestion
textFiles: 0 "controllers/portal_pe/portal_pe.c"
```